erpnext/controllers/sales_and_purchase_return.py

```diff
@@ -26,11 +26,54 @@ def validate_return(doc):
 		validate_return_against(doc)
 
 		if doc.doctype in ("Sales Invoice", "Purchase Invoice") and not doc.update_stock:
+			validate_returned_invoice_items_qty(doc)
 			return
 
 		validate_returned_items(doc)
 
 
+def validate_returned_invoice_items_qty(doc):
```

Comment:
There is already a function named `validate_quantity` for validating returned qty. Please debug that function and make the fix.